sinks/honeycomb.go

```diff
@@ -26,6 +26,7 @@ func (hs *HoneycombSink) Start() error {
 	for _, v := range hs.DropFields {
 		hs.dropFieldsMap[v] = struct{}{}
 	}
+	libhoney.UserAgentAddition = "honeycomb-opentracing-proxy"
```

Comment:
I'd love to see the version in here too, in the form "honeycomb-opentracing-proxy/1.2.3"
fine to add later if you prefer.

Reply:
Good point. Will do that in a later patch (needs change to the build process)